packages/backend/migration/1706791962000-fix-meta-disableRegistration.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-FileCopyrightText: syuilo and other misskey contributors
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+export class FixMetaDisableRegistration1706791962000 {
+    name = 'FixMetaDisableRegistration1706791962000'
+
+    async up(queryRunner) {
+        await queryRunner.query(`alter table meta	alter column "disableRegistration" set default true;`);
```

Comment:
```suggestion
        await queryRunner.query(`alter table meta alter column "disableRegistration" set default true;`);
```

tabか何かになってる？（downも同様）

Reply:
スペースに直しました